tests/func/test_init.py

```diff
@@ -64,3 +68,37 @@ def test_cli(self):
         self.assertEqual(ret, 0)
 
         self._test_init()
+
+
+class TestInitQuietShouldNotDisplayWelcomeScreen(TestGit):
+    welcome_msg_1 = boxify(
+        "DVC has enabled anonymous aggregate usage analytics.\n"
+        "Read the analytics documentation (and how to opt-out) here:\n"
+        "{blue}https://dvc.org/doc/user-guide/analytics{nc}".format(
+            blue=colorama.Fore.BLUE, nc=colorama.Fore.RESET
+        ),
+        border_color="red",
+    )
```

Comment:
This duplicates message from actual code. Any change in welcome message there will require coordinated change here.